### PR TITLE
gh-460: Refine menu items

### LINKED ISSUE #460: Menu bar refinements
# Description

Hide Templates option for now. Simply make it invisible.
Rename "SDE" to be "Requests"

In `src\app\pages\sde-main\sde-main.component.html` rename heading `Secure Data Environment` to be `Requests`

### PR DESCRIPTION
**Pull request by NigelPalmer**
closes #460

### CODE REVIEW

File: src/app/pages/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -57,26 +50,25 @@ <h2>Your current data specifications</h2>
...
         <a routerLink="/search">Search</a> or <a routerLink="/browse">browse</a> our
         catalogue to identify further data points of interest. You can also use our
         <a routerLink="/templates">data specification templates</a>, curated to represent
         common scenarios, as a starting point for your own data specifications.
+        -->
       </p>
       <p *ngIf="currentUserDataSpecifications.length === 0" class="text-center">
-        No active data specifications yet? Search the catalogue to start, or review our
-        templates.
+        No active data specifications yet? Search the catalogue to start, or review our templates.
```
Comment (abwilson23) [inline src/app/pages/dashboard/dashboard.component.html]:
```suggestion
        No active data specifications yet? Browse our catalogue to start.
```

**Review by abwilson23 — DISMISSED** [review]
If the goal is just to hide templates from the menu, then this is a pass ✅.

I've left a suggestion to remove the word 'templates' from some help text if the user doesn't have any data specifications. Additionally, you can still access the templates page by typing in the url. If you'd like to prevent that then you'll need to remove the templates path from the `pages.routes.ts` file.

File: src/app/pages/dashboard/dashboard.component.html (hunk 1 above)
Comment (NigelPalmer) [reply src/app/pages/dashboard/dashboard.component.html]: Updated as suggested.

**Comment by NigelPalmer** [comment]
Accessing the templates page via URL is fine. For now we just don't want it obviously visible.

**Review by abwilson23 — APPROVED** [review]
Approved ✅

**Pull request merged**